modules/installation-gcp-shared-vpc-config.adoc

```diff
@@ -62,16 +58,13 @@ networking:
   machineNetwork:
   - cidr: 10.0.0.0/16
 pullSecret: '{"auths": ...}'
-sshKey: ssh-ed25519 AAAA... <11>
+sshKey: ssh-ed25519 AAAA... <8>
 ----
 <1> `credentialsMode` must be set to `Passthrough` to allow the cluster to use the provided GCP service account after cluster creation. See the "Prerequisites" section for the required GCP permissions that your service account must have.
 <2> The name of the subnet in the shared VPC for compute machines to use.
 <3> The name of the subnet in the shared VPC for control plane machines to use.
-<4> Optional. If you set `createFirewallRules` to `Disabled`, you can create and manage firewall rules manually through the use of network tags. By default, the cluster will automatically create and manage the firewall rules that are required for cluster communication. Your service account must have `roles/compute.networkAdmin` and `roles/compute.securityAdmin` privileges in the host project to perform these tasks automatically. If your service account does not have the `roles/dns.admin` privilege in the host project, it must have the `dns.networks.bindPrivateDNSZone` permission.
-<5> The name of the shared VPC.
-<6> The name of the host project where the shared VPC exists.
-<7> Optional. The name of a public DNS zone in the host project. If you set this value, your service account must have the `roles/dns.admin` privilege in the host project. The public DNS zone domain must match the `baseDomain` parameter. If you do not set this value, the installation program will use the public DNS zone in the service project.
-<8> Optional. The name of the host project which contains the public DNS zone. This value is required if you specify a public DNS zone that exists in another project.
-<9> The name of the GCP project where you want to install the cluster.
-<10> Optional. If you want to manually create and manage your GCP firewall rules, you can set `platform.gcp.createFirewallRules` to `Disabled` and then specify one or more network tags. You can set tags on the compute machines, the control plane machines, or all machines.
```

Comment:
I don't know if this is the right place for it, but I see the create firewall rules was here. We need some additional documentation about the new feature for firewall rule creation. It will only apply to xpn installs so it's probably best here. The user should have permission `compute.firewalls.create` in the host project or the firewall rules will not be created (they are skipped) by the installer. This behavior mirrors the k8s behavior meaning that the k8s firewall rules will not be created either if the permissions are absent.

Reply:
Is the intent that the customer should choose whether or not to do this? IE:
1) customer obtains the `compute.firewall.create` permission in the host project so that the installer creates the firewall rules OR
2) deliberately don't obtain that permission because the customer intends to use tags or other means to create the firewall

Reply:
It is suggested that the user has these permissions. The intent was to mirror the kubernetes behavior. This ensures that either all or none of the firewall rules are created. Technically if the user does not want to create firewall rules then they can remove that permission but it is definitely not intended.